samples/sensor_monitoring/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -17,6 +17,32 @@ sid_semtech: &spi2 {
 	clock-frequency = <DT_FREQ_M(8)>;
 };
 
+/{
+ 	semtech_sx1262_gpios{
+ 		compatible = "gpio-keys";
+		semtech_sx1262_cs: cs {
+			gpios = <&gpio1 0x8 GPIO_ACTIVE_LOW>;
+			label = "semtech_sx1262 CS";
+		};
+ 		semtech_sx1262_reset_gpios: reset {
+			gpios = <&arduino_header 0 GPIO_ACTIVE_LOW>;
```

Comment:
Use gpio instead of the arduino_header alias
```suggestion
			gpios = <&gpioX 0 GPIO_ACTIVE_LOW>;
```

Reply:
OK